tests/console/microcode_update.pm

```diff
@@ -19,14 +19,17 @@ use utils 'zypper_call';
 use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
-    my ($self, $vendor) = @_;
+    my ($self, $vendor, $match) = @_;
     use_ssh_serial_console;
     # different true exit status value bash 0 perl 1
     if (script_run 'lscpu|grep -i intel') {
         $vendor = 'amd';
+        # on amd is no dmesg message which would prove or disprove microcode update
```

Comment:
I think there is message ...

```
[    3.618158] mce: Using 28 MCE banks
[    3.618216] microcode: CPU0: patch_level=0x0830101c
[    3.618225] microcode: CPU1: patch_level=0x0830101c
[    3.618235] microcode: CPU2: patch_level=0x0830101c
[    3.618238] microcode: CPU3: patch_level=0x0830101c
[    3.618251] microcode: CPU4: patch_level=0x0830101c
[    3.618265] microcode: CPU5: patch_level=0x0830101c
[    3.618278] microcode: CPU6: patch_level=0x0830101c
[    3.618291] microcode: CPU7: patch_level=0x0830101c
[    3.618307] microcode: CPU8: patch_level=0x0830101c
[    3.618319] microcode: CPU9: patch_level=0x0830101c
[    3.618331] microcode: CPU10: patch_level=0x0830101c
[    3.618343] microcode: CPU11: patch_level=0x0830101c
[    3.618359] microcode: CPU12: patch_level=0x0830101c
[    3.618371] microcode: CPU13: patch_level=0x0830101c
[    3.618383] microcode: CPU14: patch_level=0x0830101c
[    3.618395] microcode: CPU15: patch_level=0x0830101c
[    3.618411] microcode: CPU16: patch_level=0x0830101c
[    3.618424] microcode: CPU17: patch_level=0x0830101c
[    3.618435] microcode: CPU18: patch_level=0x0830101c
[    3.618447] microcode: CPU19: patch_level=0x0830101c
[    3.618463] microcode: CPU20: patch_level=0x0830101c
[    3.618475] microcode: CPU21: patch_level=0x0830101c
[    3.618486] microcode: CPU22: patch_level=0x0830101c
[    3.618498] microcode: CPU23: patch_level=0x0830101c
[    3.618514] microcode: CPU24: patch_level=0x0830101c
[    3.618526] microcode: CPU25: patch_level=0x0830101c
[    3.618540] microcode: CPU26: patch_level=0x0830101c
[    3.618552] microcode: CPU27: patch_level=0x0830101c
[    3.618568] microcode: CPU28: patch_level=0x0830101c
[    3.618580] microcode: CPU29: patch_level=0x0830101c
[    3.618592] microcode: CPU30: patch_level=0x0830101c
[    3.618603] microcode: CPU31: patch_level=0x0830101c
[    3.618620] microcode: CPU32: patch_level=0x0830101c
[    3.618633] microcode: CPU33: patch_level=0x0830101c
[    3.618645] microcode: CPU34: patch_level=0x0830101c
```

Reply:
it is there also when ucode-amd is not installed